src/aggregation/intermediate_agg_result.rs

```diff
@@ -281,6 +281,21 @@ impl IntermediateBucketResult {
                         .unwrap_or(f64::MIN)
                         .total_cmp(&right.from.unwrap_or(f64::MIN))
                 });
+
+                let is_keyed = req
+                    .as_range()
+                    .expect("unexpected aggregation, expected range aggregation")
+                    .keyed
+                    .is_some();
+                let buckets = if is_keyed {
+                    let mut bucket_map = HashMap::new();
```

Comment:
we can use with_capacity here

Reply:
fixed! https://github.com/quickwit-oss/tantivy/commit/80a1418284b140a833fe35aba688d0b14197a9b2